README.md

```diff
@@ -240,3 +240,20 @@ Make sure before you changed and open a PR to run the follow script and **paste
 
 Obs.: 1869 and 1870 is just an example of versions to be shown. Use the GO build
 number in ascending order.
+
+## Code Backup
+
+Code that has been removed from frontend but saved for possible later use.
+They are stored as individual branches of the frontend repo.
+It is recommended to rebase off of master before implementation.
```

Comment:
Since rebasing off master before implementation is/should be standard practice, not sure if it needs to be specified here.

Reply:
@margOnline I'll remove the recommendation to rebase. thought it would be best to be specific but no biggie. In terms of where the documentation should live, we discussed this with @willhall88 and @tomas-stefano in the PR that removed and backed up the code. We agreed the readme was the best place to store this information. 

https://github.com/moneyadviceservice/frontend/pull/1871

Reply:
My concern is that the info in the repo readme could then creep beyond its original purpose and the information would get lost. readmes generally contain information such as [this template](https://gist.github.com/PurpleBooth/109311bb0361f32d87a2) or [this template](https://gist.github.com/jxson/1784669).

One suggestion would be to change the title of the `Backup code` section to `Features` and link to a separate markup files, as is done in [technical docs](https://github.com/moneyadviceservice/technical-docs), where Engines section lists the different engines.

I think the TP card should also include a link to the feature branch.

Reply:
@willhall88 @tomas-stefano @davidtrussler @johnplayer1982  What do you guys think about @margOnline 's suggestion? I believe we had consensus that the solution put forth in this PR was fit for purpose and would be sufficient if this functionality needs to be brought back in the future.